hepcrawl/extractors/iop_parser.py

```diff
@@ -107,7 +107,7 @@ def parse_node(self, response, node):
     def get_arxiv_eprints(self, node):
         arxiv_eprints = []
 
-        arxivs_raw = node.xpath("//article-id[@pub-id-type='arxiv']/text()")
+        arxivs_raw = node.xpath("//custom-meta-group/custom-meta/meta-value/text()")
```

Comment:
hmm are we sure it's on the same xml file?

Reply:
yes, I was testing just with the same xml file

Reply:
* tests